packages/compass-query-bar/src/components/query-bar/query-bar.module.less

```diff
@@ -95,3 +95,8 @@
 	margin-left: 5px;
 	width: 28px;
 }
+
+.query-option-toggle-row {
+  display: flex;
+  flex-direction: row;
```

Comment:
Because the first row now has both input and button as children of the flex container and input is not taking the whole width, the top border that is part of the `QueryOption` is now awkwardly cuts right before the button in the input:

<img width="587" alt="image" src="https://user-images.githubusercontent.com/5036933/133798752-ac42c0ba-bcc8-47fa-9dcf-3987a9c2f1ef.png">

We should address this regression somehow before merging. I think even removing border from all the `QueryOption` elements in the first row instead of trying to fix the border there should be fine. @Anemy wdyt?

Reply:
Good eye, totally missed that. I think the border adds a nice bit of depth to the component, so it would be nice to keep it if possible. I think we could remove it from the query option in the first row and add it to the new container which has both elements.